src/main/java/com/softserveinc/ita/vsaroz/pageobjects/JoinFormPage.java

```diff
@@ -0,0 +1,27 @@
+package com.softserveinc.ita.vsaroz.pageobjects;
+
+import com.softserveinc.ita.vsaroz.models.User;
+
+import static com.softserveinc.ita.vsaroz.utils.runners.ElementUtils.setText;
+
+public class JoinFormPage {
+
+    protected final String homepageJoinButtonElement = ("//a[@href='/join']");
```

Comment:
after your changes it stopped being an element, currently, it's "path", delete brackets ()